gradle/libs.versions.toml

```diff
@@ -37,7 +37,7 @@ paparrazzi = "1.3.1"
 coil = "2.4.0"
 window = "1.1.0"
 commonmark = "0.21.0"
-activity-compose = "1.7.2"
+activity-compose = "1.9.3"
```

Comment:
I had to update this because the AS app generation was expecting a newer version. I checked the changes and didn't see anything worth calling out...